lib/senders/templates/verification_reminder_first.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <title>{{t "Firefox Accounts"}}</title>
```

Comment:
If this is the email subject like. Replace it by:

> Reminder: Confirm your email to activate your Firefox Account

Similarly to the email body, I'm intentionally repeating the call to action in the subject line.

Reply:
(fwiw, the email subject was previously "Hello again" for this one)

Reply:
OK thanks. I stand by my suggestion. Let's be clear with the user.